### PR TITLE
Use right naming for ZEIT

### PR DESCRIPTION
**Pull request by timneutkens**

### CODE REVIEW

**Comment by helfer** [comment]
Thanks @timneutkens! Happy to merge this once https://github.com/zeit/micro/pull/274 is merged :wink:

File: README.md [hunk 1]
```diff
@@ -166,9 +166,9 @@ var server = require("graphql-server-lambda");
 exports.handler = server.graphqlLambda({ schema: myGraphQLSchema });
 ```
 
-### Zeit Micro
+### ZEIT Micro
 
-Micro function requires the micro module
+Requires the [Micro](github.com/zeit/micro) module
```
Comment (j-f1) [inline README.md]: This is a relative URL; you probably want `[Micro](https://github.com/zeit/micro)`.

Comment (timneutkens) [inline README.md]: Ha! Fail 😅

Comment (timneutkens) [inline README.md]: Fixed!

**Comment by timneutkens** [comment]
@helfer it's merged 😄

**Comment by helfer** [comment]
Perfect, thanks a lot @timneutkens!

**Pull request merged**